README.md

```diff
@@ -1,35 +1,70 @@
-[Website](https://www.cortex.dev) • [Slack](https://community.cortex.dev) • [Docs](https://docs.cortex.dev)
```

Comment:
I think I still like having these links at the top (or they could be moved to after the logo or to where "Get started" is). We can discuss